lexicons/com/atproto/repo/getBlob.json

```diff
@@ -0,0 +1,20 @@
+{
+  "lexicon": 1,
+  "id": "com.atproto.repo.getBlob",
+  "defs": {
+    "main": {
+      "type": "query",
+      "description": "Fetch a blob.",
+      "parameters": {
+        "type": "params",
+        "required": ["cid"],
+        "properties": {
+          "cid": {"type": "string", "description": "The CID of the blob to fetch"}
+        }
+      },
+      "output": {
+        "encoding": "application/octet-stream"
```

Comment:
Even if we choose to serve the files as application/octet-stream, I'd still be +1 to leave it open `*/*` in the lex spec.  My basic argument is knowledge is power haha— I don't think you really lose anything by allowing hosts to say what they consider the bytes that they are serving to represent.